packages/now-cli/test/integration.js

```diff
@@ -1386,28 +1386,29 @@ test('deploy multiple static files with custom scope', async t => {
   t.is(content.files.length, 3);
 });
 
-test('deploying a file should fail', async t => {
+test('deploying a file should not show prompts and display deprecation', async t => {
   const file = fixture('static-single-file/first.png');
 
-  const { stdout, stderr, exitCode } = await execa(
-    binaryPath,
-    [file, '--public', '--name', session, ...defaultArgs, '--confirm'],
-    {
-      reject: false,
-    }
-  );
+  const output = await execute([file], {
+    reject: false,
+  });
 
-  console.log(stderr);
-  console.log(stdout);
-  console.log(exitCode);
+  const { stdout, stderr, exitCode } = output;
 
   // Ensure the exit code is right
-  t.is(exitCode, 1);
-  t.true(
-    stderr
-      .trim()
-      .endsWith('The path you are trying to deploy is not a directory.')
-  );
+  t.is(exitCode, 0, formatOutput(output));
```

Comment:
Should we also verify that `.now` does not exist?

Reply:
Tackled in 4bd414e